plugins/trace-activation/create_entryspan_intercepter.go

```diff
@@ -27,13 +27,20 @@ type CreateEntrySpanInterceptor struct {
 }
 
 func (h *CreateEntrySpanInterceptor) BeforeInvoke(invocation operator.Invocation) error {
-	operationName := invocation.Args()[0].(string)
-	var extractor func(headerKey string) (string, error) = invocation.Args()[1].(trace.ExtractorRef)
-	s, err := tracing.CreateEntrySpan(operationName, extractor)
-	invocation.DefineReturnValues(s, err)
 	return nil
 }
 
 func (h *CreateEntrySpanInterceptor) AfterInvoke(invocation operator.Invocation, result ...interface{}) error {
+	operationName := invocation.Args()[0].(string)
+	var extractor func(headerKey string) (string, error) = invocation.Args()[1].(trace.ExtractorRef)
+	s, err := tracing.CreateEntrySpan(operationName, extractor)
```

Comment:
The error message cannot be ignored, we should re-write the result when the error is not nil.